edxval/api.py

```diff
@@ -277,17 +278,62 @@ def get_videos_for_ids(
     return (VideoSerializer(video).data for video in videos)
 
 
-def get_video_info_for_course_and_profile(course_id, profile_name):
+def get_video_info_for_course_and_profiles(course_id, profiles):
     """
-    Returns a dict mapping profiles to URLs.
+    Returns a dict of edx_video_ids with a dict of profiles.
 
     If the profiles or video is not found, urls will be blank.
+
+    Args:
+        course_id (str): id of the course
+        profiles (list): list of profile_names
+    Returns:
+        return_dict (dict): Returns all the profiles attached to a specific
+        edx_video_id
+        {
+            edx_video_id: {
+                profile_name: {
+                    'url': url of the encoding
+                    'duration': length of the video in seconds
+                    'file_size': size of the file in bytes
+                },
+            },
+        }
+    Example:
+        Given two videos with two profiles each in course_id 'test_course':
+        >>> get_video_info_for_course_and_profiles('test-course', ['mobile', 'desktop''])
+        {
+            u'super-soaker': {
+                u'mobile': {
+                    'url': u'http: //www.meowmix.com',
```

Comment:
Use example.com for example URLs. Also, the example should distinguish between the URLs for the two videos. I suggest something like `http://example.com/super-soaker-mobile.mp4`